observability/otel/otel-rn.adoc

```diff
@@ -12,9 +12,39 @@ You can use the {OTELName} xref:otel-forwarding-telemetry-data.adoc#otel-forward
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
-// git rebase marker 1
+[id="otel_3-4_{context}"]
+== Release notes for {OTELName} 3.4
 
-// git rebase marker 2
+The {OTELName} 3.4 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/opentelemetry-operator-bundle/615618406feffc5384e84400/history[{OTELOperator} 0.113.0].
+
+The {OTELName} 3.4 is based on the open source link:https://opentelemetry.io/docs/collector/[OpenTelemetry] release 0.113.0.
+
+[id="otel_3-4_technology-preview-features_{context}"]
+=== Technology Preview features
+
+This update introduces the following Technology Preview features:
+
+* OTLP JSON File Receiver
```

Comment:
You mean like this?
```
OpenTelemetry Protocol (OTEL) JavaScript Object Notation (JSON) File Receiver
```
I had a look at the OCP docs, and JSON not normally expanded. For example here: https://docs.openshift.com/container-platform/4.17/observability/network_observability/netobserv_cli/netobserv-cli-using.html
I guess it is because IBM Style rules that it is unnecessary:
"_Do not spell out an abbreviation that is commonly known to the target audience. Commonly known terms vary by audience; each writing and editing team must determine which abbreviations are familiar to the audience.
Correct example: When known to the audience: DOS, HTML, HTTP, PDF, TCP/IP_"

Reply:
@kalexand-rh, do we expand JSON and similar terms in the OCP docs?

Reply:
Sorry for the confusion. Just the OTPL abbreviation is fine, @max-cx. You don't need to expand JSON.

Reply:
Done.